src/php/Analyzer/TupleSymbol/QuoteSymbol.php

```diff
@@ -6,15 +6,20 @@
 
 use Phel\Ast\QuoteNode;
 use Phel\Exceptions\AnalyzerException;
+use Phel\Lang\Symbol;
 use Phel\Lang\Tuple;
 use Phel\NodeEnvironment;
 
 final class QuoteSymbol
 {
     public function __invoke(Tuple $tuple, NodeEnvironment $env): QuoteNode
     {
+        if (!($tuple[0] instanceof Symbol && $tuple[0] == Symbol::NAME_QUOTE)) {
```

Comment:
It should be `$tuple[0]->getName() === Symbol::NAME_QUOTE`

Reply:
Uups. Of course :)

Reply:
Use the `getName()` function instead of string casting.

Reply:
Apply suggestion and update the names of the tests. I think now is much better!